server/sonar-server/src/main/java/org/sonar/server/authentication/CookieUtils.java

```diff
@@ -40,4 +41,19 @@ private CookieUtils() {
       .filter(cookie -> cookieName.equals(cookie.getName()))
       .findFirst();
   }
+
+  public static Cookie createCookie(String name, @Nullable String value, boolean httpOnly, int expiry, HttpServletRequest request){
+    Cookie cookie = new Cookie(name, value);
+    // Path is set "/" in order to allow rails to be able to remove cookies
+    // When logout when be implemented in Java (SONAR-7774), following line should be replaced by cookie.setPath(request.getContextPath()"/");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)